tests/bench/locomo_baseline.py

```
@@ -0,0 +1,534 @@
+load_dotenv(bench_dir / ".env")
+
+# OpenRouter model format for baseline testing
+MODEL_BEING_TESTED = "anthropic/claude-haiku-4.5"
+
+
+class LoCoMoBaselineRunner:
+    """
+    Executes LoCoMo benchmark tests directly against Claude.
+    """
+
+    def __init__(self):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)